conmon-rs/server/src/init.rs

```diff
@@ -1,12 +1,16 @@
 use anyhow::{bail, Result};
-use gettextrs::{setlocale, LocaleCategory};
+use libc::{setlocale, LC_ALL};
 use log::info;
+use std::ffi::CString;
 use std::fs::File;
 use std::io::{ErrorKind, Write};
 
 /// Unset the locale for the current process.
-pub fn unset_locale() {
-    setlocale(LocaleCategory::LcAll, "");
+pub fn unset_locale() -> Result<()> {
+    unsafe {
+        setlocale(LC_ALL, CString::new("")?.as_ptr());
+    }
+    Ok(())
```

Comment:
Do we need to return this since we aren't actually propagating any errors?

Reply:
It does get propagated in this case.